packages/engine/src/assets/classes/AssetLoader.ts

```diff
@@ -211,7 +211,7 @@ const assetLoadCallback = (url: string, assetType: AssetType, onLoad: (response:
   onLoad(asset)
 }
 
-const getAbsolutePath = (url) => (isAbsolutePath(url) ? url : Engine.publicPath + url)
+export const getAbsolutePath = (url) => (isAbsolutePath(url) ? url : Engine.publicPath + url)
```

Comment:
as part of our ongoing refactor efforts towards modularity and extensibility, we should not be exporting these functions, but using the module exporter `AssetLoader.getAbsolutePath`  as per the style guide https://github.com/XRFoundation/XREngine/wiki/Style-Guide